user-guide/Cloud_Platform/Catalog/Tutorials/Tutorial_Register_Catalog_Item.md

```diff
@@ -0,0 +1,114 @@
+---
+uid: Tutorial_Register_Catalog_Item
+---
+
+# Registering a new connector in the Catalog
+
+This tutorial demonstrates how to add a new Catalog item using the [Catalog API](xref:Register_Catalog_Item), using an [example connector](https://github.com/SkylineCommunications/SLC-C-Example_Rates-Custom).
+
+For ease of use, [Postman](https://www.postman.com/) will be used to execute the register API calls. You can also integrate these into your own workflow afterwards, e.g by using them in a GitHub action.
+
+## Prerequisites
+
+- An [organization key](xref:Managing_DCP_keys#organization-keys) or account with the *Owner* role in order to access/create organization keys.
+
+  > [!TIP]
+  > See [Changing the role of a dataminer.services user](xref:Changing_the_role_of_a_DCP_user)
+
+## Overview
+
+- [Step 1: Register the Catalog item URL](#step-1-register-the-catalog-item-url)
+- [Step 2: Configure the authentication header](#step-2-configure-the-authentication-header)
+- [Step 3: Register the Catalog item body](#step-3-register-the-catalog-item-body)
+- [Step 4: Register the Catalog item](#step-4-register-the-catalog-item)
+
+## Step 1: Register the Catalog item URL
+
+Add a new request using an HTTP method of type **PUT** and the following URL: `https://api.dataminer.services/api/key-catalog/v1-0/catalog/register`
+
+![Register item HTTP URL](~/user-guide/images/tutorial_catalog_registration_item_url.png)
```

Comment:
The text in this screenshot is so small that it's barely readable. Could you replace this with a better screenshot?

Reply:
updated images, also for version tutorial